templates/index.ts

```diff
@@ -4,6 +4,8 @@ export { default as appsmith } from "./appsmith";
 export { default as bookstack } from "./bookstack";
 export { default as directus } from "./directus";
 export { default as domainmod } from "./domainmod";
+export { default as drone } from "./drone";
+export { default as dronerunner } from "./dronerunner";
```

Comment:
Let's name the folder `drone-runner` and the import `droneRunner`. It should become `export { default as droneRunner } from "./drone-runner";`